### PR TITLE
UPDATE: Tooltip accessibility

### PR DESCRIPTION
**Pull request by filipdanisko**
- Tooltip is now able to be open with focus
- Addiing aria tags
- Adding helper function to generate randomid<br/><br/><br/><url>LiveURL: https://orbit-components-update-tooltip-a11y.surge.sh</url>

### CODE REVIEW

File: src/Tooltip/index.js [hunk 1]
```diff
@@ -437,13 +442,16 @@ class Tooltip extends React.PureComponent<Props, State> {
           onFocus={this.handleIn}
           onBlur={this.handleOut}
           ref={this.container}
+          aria-describedby={this.tooltipId}
+          tabIndex="0"
         >
           {children}
         </StyledTooltipChildren>
         <Portal element="tooltips">
           <StyledTooltip role="tooltip" data-test={dataTest}>
```
Comment (vepor) [inline src/Tooltip/index.js]: Tooltip has now `role` twice. Is in intentional?

File: src/utils/randomId/index.flow.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+// @flow
+export type Random = (value: string) => string;
+
+declare export default function randomId(name: string): string;
```
Comment (vepor) [inline src/utils/randomId/index.flow.js]:
```
export type RandomID = (value: string) => string;

declare export default RandomID;
```

File: src/utils/randomId/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// @flow
+import type { Random } from "./index.flow";
+
+const randomId: Random = (name: string) => {
+  return name + Math.floor(Math.random() * 1000000000) + Math.floor(Math.random() * 1000000000);
```
Comment (vepor) [inline src/utils/randomId/index.js]: 😅 won't be `thousand` enough?

Comment (vepor) [reply src/utils/randomId/index.js]: Also, plz `randomID` 🙏

Comment (filipdanisko) [reply src/utils/randomId/index.js]: 1000 is a length of 3, that's not sufficient I had it originally on that, but after running a benchmark, it returned match multiple times on 1000 items. So I cracked that up, but maybe we don't need an addition of extra random number in that case?

Also, randomID is not proper camel-case

File: src/Tooltip/index.js (hunk 1 above)
Comment (filipdanisko) [reply src/Tooltip/index.js]: Good catch!

File: src/Tooltip/index.js [hunk 4]
```diff
@@ -32,14 +32,14 @@ import {
 import { isAlignCenter, isAlignEnd, isAlignStart } from "./helpers/isAlign";
 import { DEVICES_WIDTH } from "../utils/mediaQuery/consts";
 import tooltipPadding from "./helpers/tooltipPadding";
+import generateRandomId from "../utils/randomId";
```
Comment (vepor) [inline src/Tooltip/index.js]: Always use the same name as the file has 🙏

File: src/Tooltip/index.js [hunk 5]
```diff
@@ -437,13 +442,16 @@ class Tooltip extends React.PureComponent<Props, State> {
           onFocus={this.handleIn}
           onBlur={this.handleOut}
           ref={this.container}
+          aria-describedby={this.tooltipId}
+          tabIndex="0"
```
Comment (vepor) [inline src/Tooltip/index.js]: https://github.com/kiwicom/orbit-components/pull/945#discussion_r270847115

**Pull request merged**